platform-validation/scripts/compatibility.sh

```diff
@@ -12,6 +12,30 @@ ARCH=""
 #  Text Formatting
 #  Derived from : https://github.com/moby/moby/blob/master/contrib/check-config.sh
 # ------------------------------------------------------------------------------
+
+possibleConfigs="
+	/proc/config.gz
+	/boot/config-$(uname -r)
+	/usr/src/linux-$(uname -r)/.config
+	/usr/src/linux/.config
+"
+
+if [ $# -gt 0 ]; then
+	CONFIG="$1"
+else
+	: "${CONFIG:=/proc/config.gz}"
+fi
+
+if ! command -v zgrep > /dev/null 2>&1; then
+	zgrep() {
+		zcat "$2" | grep "$1"
+	}
+fi
+
+is_set() {
```

Comment:
Do you plan to use this?

Reply:
I think I need some follow up investigation, before I can start using them. But yes, i do intend to use them